sdk/python/feast/infra/registry_stores/sql.py

```diff
@@ -300,13 +353,14 @@ def apply_data_source(
     def apply_feature_view(
         self, feature_view: BaseFeatureView, project: str, commit: bool = True
     ):
-        # TODO(achals): Stream feature views need to be supported.
         if isinstance(feature_view, FeatureView):
             fv_table = feature_views
         elif isinstance(feature_view, OnDemandFeatureView):
             fv_table = on_demand_feature_views
         elif isinstance(feature_view, RequestFeatureView):
             fv_table = request_feature_views
+        elif isinstance(feature_view, StreamFeatureView):
```

Comment:
this actually doesn't work since streamfeatureview is actually an instance of featureview(it inherits feature view) Either streamfeatureview has to go first or the first isinstance has to assert that it's not a batch or streamfeatureview.

Reply:
Hmmm why does StreamFeatureView subclass FeatureView?

Reply:
done